src/NUnitFramework/framework/Constraints/EqualityAdapter.cs

```diff
@@ -84,9 +91,9 @@ public override bool AreEqual(object x, object y)
             }
         }
 
-        #endregion
+#endregion
 
-        #region Nested IEqualityComparer Adapter
+#region Nested IEqualityComparer Adapter
```

Comment:
There are other unindented regions in this file. I'm assuming you'd like me to fix those as well.